0004/README.md

```diff
@@ -0,0 +1,416 @@
+
+## Prior art
+
+CNCF's Certified Kubernetes Software Conformance -
+[https://www.cncf.io/certification/software-conformance/](https://www.cncf.io/certification/software-conformance/)
+
+## Proposed changes
+
+### Overview
+
+The OpenSSF maintains a SLSA Conformance badge and registry of build platforms
+and auditors that take part in the SLSA Conformance Program. Participating in
+the program allows build platforms to use the SLSA Conformance badge when
+promoting themselves. Any build platform or auditor is free to participate in
+the SLSA Conformance Program provided they are willing to meet its requirements
+and follow its terms.  
+
+The SLSA community provides a security questionnaire for build platforms. This
+questionnaire is the minimum standard for evidence of SLSA conformance.
+Verifiers and auditors are encouraged to seek evidence beyond the questionnaire
+when making their trust determinations. For a draft of the questionnaire, see
+Appendix A.
+
+### Build platform participation
+
+There are two tiers for a build platform to take part in the program:
+self-attestation and third-party audit. Self-attestation is a weaker proof of
+conformance than a third-party audit, and is most useful while bootstrapping the
+conformance program before auditors are available.
+
+#### Tier one: Self-attestation
+
+The self-attestation tier requires that a build platform:
+
+-  Publish answers to the questionnaire on a domain registered to their
+    organization. They may additionally publish further proof of SLSA
+    conformance (e.g. architecture documentation). This information must be
+    updated at least once every twelve months and any time there is an
+    architectural change to the build platform.
+-  Publish any materials needed to verify provenance signed by the build
+    system on a domain registered to their organization.
+-  Publish vulnerability disclosures for all security incidents that
+    potentially impact the integrity of artifacts or provenance generated by
+    the build system.
+-  Sign and follow the SLSA Conformance Program terms.
+
+The OpenSSF operates this tier through a community-maintained repository of
+build platforms. The maintainers of this repository will be drawn from members
+of the SLSA community who are spread across the industry.  
+In order to participate in this tier, build platforms should 
+
+1. Learn about the SLSA Specification and its requirements.
+1. Create a PR for the repository containing
+    1. Name of the build platform
+    1. Build ID
+    1. Desired SLSA levels
+    1. List of URIs that resolve to proof of SLSA conformance
+    1. List of URIs that resolve to materials for verifying provenance
+    1. List of URIs that resolve to the build platform's public
+        vulnerability disclosures
+
+1. Complete the terms and conditions form.
+1. Send the PR for review.
+
+The community will review the PR and merge it when a quorum of maintainers
```

Comment:
Can you link to the list of maintainers that would vote on it?

Reply:
I don't have a list and am not entirely sure the best way to get one. I was thinking maybe volunteers from the current SLSA contributors with some limit to the number who can be at one company, but I'm open to suggestions.

Reply:
Would this be considered everyone with write access to the repository? What is considered a quorum? Is it a simple majority? Is there a minimum number that need to be approving? Some specific subset of maintainers?

> Quorum:
> the minimum number of officers or members of a body that is required to be present at a given meeting (as to transact business)

Reply:
We won't really know how many maintainers there are until we're able to gather volunteers, so let's go with 2+. We can always change it if that doesn't work.